helm/charts/keto/templates/deployment.yaml

```
@@ -52,6 +52,12 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- if .Values.deployment.extraInitContainers}}
+      initContainers:
+        {{- if .Values.deployment.extraInitContainers }}
+{{ tpl .Values.deployment.extraInitContainers . | indent 8 }}
```

```suggestion
{{- tpl .Values.deployment.extraInitContainers . | nindent 8 }}
```
we prefer to use `nindent` as it allows for indentation of the functions within templates :)

---

Ah well, I copied everything from kratos....but ok, I'll do it.

---

Done, I also fixed the equivalent `$extraContainers` later in the file.